libkbfs/interfaces.go

```diff
@@ -925,13 +930,17 @@ type DiskBlockCache interface {
 	// Delete deletes some blocks from the disk cache.
 	Delete(ctx context.Context, blockIDs []kbfsblock.ID) (numRemoved int,
 		sizeRemoved int64, err error)
-	// UpdateMetadata updates the LRU time to Now() for a given block.
+	// UpdateMetadata updates metadata for a given block in the disk cache.
 	UpdateMetadata(ctx context.Context, blockID kbfsblock.ID,
-		hasPrefetched bool) error
+		hasPrefetched, donePrefetch bool) error
```

Comment:
Yeah I think that would be a useful thing to know, while I can't really think of a use for the current semantics.  If it's going to take a while though, it might not be worth blocking this PR on.  But I think we should have it in place before turning it on for non-admins.